fasttrips/FastTrips.py

```diff
@@ -14,6 +14,8 @@
 """
 import os, sys
 
+from functools import partial
```

Comment:
I don't see where we need to create a partial function here or anywhere?  Am I missing?